### DISCUSSION TITLE
Support for Apache Cassandra 4.1.6

### DISCUSSION THREAD

**Discussion by vanessagits** [k8ssandra/management-api-for-apache-cassandra#526] [Q&A] [answered]
Hi Team,

Is there a target date for updated version support for Apache Cassandra 4.1.6?
Note: I see that Apache Cassandra 4.1.6 change CASSANDRA-19534 updates the Dispatcher.processRequest method requiring changes in reflection for invoking  processRequestMethod in UnixSocketServer41x.java.

**Comment by emerkle826** [comment, marked as answer]
@vanessagits Thanks for the note. I'll create a ticket to get 4.1.6 added. I don't have an ETA yet on when it will be out, but I'll try to get to it soon.

**Reply by vanessagits** [reply]
Thanks @emerkle826 for the ticket: https://github.com/k8ssandra/management-api-for-apache-cassandra/issues/527

**Comment by emerkle826** [comment]
@vanessagits Cassandra 4.1.6 was added yesterday as of the v0.1.85 release.

**Reply by vanessagits** [reply]
Thank you!
